setupenv.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+unset PYTHONPATH
+python3.7 -m venv ./env
```

Comment:
I think in `setup.py` we say 3.8 is our newest supported py release.

Reply:
I was using Python 3.8 and 3.9 virtual environments for development, but then I ran into a behavior change in the way context managers were mocked. My mock worked fine in 3.8, but the Python 3.7.2 used by CircleCI failed.

Since then, I've been trying to stick to 3.7 so that the development matches what CircleCI will accept.

Maybe I should look into upgrading CircleCI to use 3.8 or 3.9? :smile:

Reply:
That's a tough scenario. On one hand, if we say we support up to 3.8, we should test it with 3.8. On the other hand, if the testing framework itself doesn't support 3.8, it's a hard call.

What would be nice is to run all the tests programmatically with each version that we claim to support in `setup.py`, which I believe is 3.6 - 3.8.

The last time I looked into 3.9, it didn't have widespread PyPI support at least not explicitly.

Reply:
resolved by https://github.com/WIPACrepo/wipac-dev-tools/blob/main/.github/workflows/try-setup-install.yml